client/src/views/cityForecastView/CityForecastView.jsx

```diff
@@ -1,7 +1,16 @@
 import React from 'react';
+import CityForecast from '../../layouts/CityForecast/CityForecast';
+import DailyForecasts from '../../layouts/dailyForecasts/DailyForecasts';
+import HourlyForecasts from '../../layouts/hourlyForecasts/HourlyForecasts';
 
-function CityForecastView() {
-  return <div></div>;
+function CityForecastView({ theme }) {
+  return (
+    <div>
```

Comment:
if you don't really need this div - then use Fragment. We don't need extra DOM nodes at the end of the day